frontend/src/components/EmbeddedPDF.tsx

```diff
@@ -17,90 +17,90 @@ const EmbeddedPDF = ({ document, passageIndex = null }: TProps) => {
   const { createPDFClient, passageIndexChangeHandler } = useMemo(() => usePDFPreview(document), [document]);
 
   // TODO: refactor and enable annotation highlighting
-  const annotationManagerConfig = {
-    showToolbar: false,
-    showCommentsPanel: false,
-    downloadWithAnnotations: true,
-    printWithAnnotations: true,
-  };
+  // const annotationManagerConfig = {
```

Comment:
Do we need this commented out code still in the file?

Reply:
It is related to the passage highlighting work that Paula started, I'll pull it out and store it separately.